dev/generate_bom.py

```diff
@@ -125,6 +127,38 @@ def write_docker_version_files(self):
       with open(config_file, 'w') as cfg:
         cfg.write(docker_tag)
 
+  def generate_changelog(self):
+    """Generate a release changelog and write it to a file.
+
+    The changelog contains a section per microservice that describes the
+    changes made since the last Spinnaker release. It also contains the
+    version information as well.
+    """
+    changelog = 'Spinnaker {0}\n\n'.format(self.__toplevel_version)
```

Comment:
changelog shoudl be a [] that you append lines to.
I guess you can append a blank.
You should probably informally sign it with a date and tool so when someone has it, they know where it came from.